sig-governance.md

```diff
@@ -46,98 +46,14 @@ Guidelines for drafting a SIG Charter can be found [here](/committee-steering/go
-* kubernetes-sig-foo-misc
-* kubernetes-sig-foo-test-failures
-* kubernetes-sig-foo-bugs
-* kubernetes-sig-foo-feature-requests
-* kubernetes-sig-foo-proposals
-* kubernetes-sig-foo-pr-reviews
-* kubernetes-sig-foo-api-reviews
-
-Example:
-
-* kubernetes-sig-onprem
-* kubernetes-sig-onprem-misc
-* kubernetes-sig-onprem-test-failures
-* kubernetes-sig-onprem-bugs
-* kubernetes-sig-onprem-feature-requests
-* kubernetes-sig-onprem-proposals
-* kubernetes-sig-onprem-pr-reviews
-* kubernetes-sig-onprem-api-reviews
-
-#### **GitHub users creation**
-
-Create the GitHub users at [https://github.com/join](https://github.com/join), using the name convention below.
-
-As an e-mail address, please, use the Google Group e-mail address of the respective Google Group, created before (i.e. - for user ‘k8s-mirror-foo-misc’ use ‘[kubernetes-sig-foo-misc@googlegroups.com](mailto:kubernetes-sig-foo-misc@googlegroups.com)’). After creating the GitHub users, please, add these users to the Kubernetes organization. If you don't have enough permissions to do that (by default, you don't), please request **@idvoretskyi** (backup person - **@sarahnovotny**) to help you with this. If GitHub contacts you about having too many robot accounts, please let us know.
-
-
-Name convention:
-
-* k8s-mirror-foo-misc 
-* k8s-mirror-foo-test-failures
-* k8s-mirror-foo-bugs
-* k8s-mirror-foo-feature-requests
-* k8s-mirror-foo-proposals
-* k8s-mirror-foo-pr-reviews
-* k8s-mirror-foo-api-reviews
-
-There is no need for a k8s-mirro-foo user.
-
-Example:
-
-* k8s-mirror-onprem-misc
-* k8s-mirror-onprem-test-failures
-* k8s-mirror-onprem-bugs
-* k8s-mirror-onprem-feature-requests
-* k8s-mirror-onprem-proposals
-* k8s-mirror-onprem-pr-reviews
-* k8s-mirror-onprem-api-reviews
-
-NOTE: We have found that Github's notification autocompletion finds the users before the corresponding teams. This is the reason we recommend naming the users `k8s-mirror-foo-*` instead of `k8s-sig-foo-*`. If you previously created users named `k8s-sig-foo-*`, we recommend you rename them.
-
-#### **Create the GitHub teams**
-
-Create the GitHub teams at [https://github.com/orgs/kubernetes/new-team](https://github.com/orgs/kubernetes/new-team), using the name convention below. Please, add the GitHub users (created before) to the GitHub teams respectively.
-
-Name convention:
-
-* sig-foo-misc 
-* sig-foo-test-failures
-* sig-foo-bugs
-* sig-foo-feature-requests
-* sig-foo-proposals
-* sig-foo-pr-reviews
-* sig-foo-api-reviews
-
-Note that there should not be a sig-foo team. We want to encourage contributors to select the most appropriate team to notify.
-
-Example:
-
-* sig-onprem-misc
-* sig-onprem-test-failures
-* sig-onprem-bugs
-* sig-onprem-feature-requests
-* sig-onprem-proposals
-* sig-onprem-pr-reviews
-* sig-onprem-api-reviews
+- Each SIG should have one discussion groups, and a number of groups for mirroring relevant github notifications;
```

Comment:
Each SIG [should|must] have _two_ discussion groups**

we should pull `, and a number of groups for mirroring relevant github notifications;`, as we're removing the suggestion to create the mirror GH accounts in this PR. The mirror Google Groups are effectively non-functional w/o the accompanying GH mirrors.